GlobalTestHelpers/MockAppModule.swift

```diff
@@ -148,6 +150,12 @@ class MockAppModule:
         tabCoordinator.asAnyCoordinator()
     }
 
+    func makeTwoFactorNoticeNoticeCoordinator(
```

Comment:
Was looking around and noticed this typo 😄

Reply:
Good catch! Thanks!